heft-plugins/heft-jest-plugin/src/JestPlugin.ts

```diff
@@ -281,6 +286,9 @@ export default class JestPlugin implements IHeftTaskPlugin<IJestPluginOptions> {
       results: jestResults
     } = await runCLI(jestArgv, [buildFolderPath]);
 
+    // unset the NODE_ENV
+    delete process.env.NODE_ENV;
```

Comment:
This should only be cleared if it was set before.

This also may interfere with other Heft tasks that run in parallel with Jest.

Reply:
@iclanton and @octogonz can you please check my last commit? 
I took your advice and implemented this functionality using the plugin option.